resource-service/src/main/java/uk/gov/gchq/palisade/service/resource/repository/AbstractReactiveRepositoryRedisAdapter.java

```diff
@@ -312,38 +339,40 @@ public SerialisedFormatRepositoryAdapter(final ReactiveRedisTemplate<String, Ser
         @Override
         @NonNull
         public <S extends SerialisedFormatEntity> Mono<S> save(final @NonNull S entity) {
-            return this.setOps.add(this.table + KEY_SEP + entity.getSerialisedFormat(), entity.getId())
-                    .then(super.saveDefault(entity));
+            return this.setOps.add(this.table + SEPARATOR + entity.getSerialisedFormat(), entity.getId())
+                    .then(this.redisTemplate.expire(this.table + SEPARATOR + entity.getSerialisedFormat(), ttl))
+                    .then(this.saveDefault(entity));
         }
 
         @Override
         public Flux<SerialisedFormatEntity> findAllBySerialisedFormat(final String serialisedFormat) {
-            return this.setOps.members(this.table + KEY_SEP + serialisedFormat)
+            return this.setOps.members(this.table + SEPARATOR + serialisedFormat)
                     .flatMap(this::findById);
         }
 
         @Override
         @NonNull
         public Mono<Void> deleteById(final @NonNull String key) {
             return this.findById(key)
-                    .map(entity -> this.setOps.remove(this.table + KEY_SEP + entity.getSerialisedFormat()))
-                    .then(super.deleteByIdDefault(key));
+                    .map(entity -> this.setOps.remove(this.table + SEPARATOR + entity.getSerialisedFormat()))
+                    .then(this.deleteByIdDefault(key));
         }
     }
 
     /**
      * A class to allow {@link TypeRepository}s to be stored in a reactive repository
      */
     public static class TypeRepositoryAdapter extends AbstractReactiveRepositoryRedisAdapter<TypeEntity, String> implements TypeRepository {
+        private static final String SEPARATOR = KEY_SEP + TYPE_KEYSPACE + KEY_SEP;
```

Comment:
I'd rename this to `TYPE_SEP` or similar

Reply:
Renamed